op-node/rollup/driver/driver.go

```diff
@@ -82,8 +81,18 @@ type Network interface {
 	PublishL2Payload(ctx context.Context, payload *eth.ExecutionPayload) error
 }
 
+type AltSync interface {
+	// RequestL2Range informs the sync source that the given range of L2 blocks is missing,
+	// and should be retrieved from any available alternative syncing source.
```

Comment:
```suggestion
	// and should be retrieved from any available alternative syncing source.
	// The start of the range is inclusive, the end is exclusive.
```

Reply:
This change didn't seem to actually get applied either.